src/OpenTelemetry.Contrib.Instrumentation.GrpcCore/ServerTracingInterceptorOptions.cs

```diff
@@ -0,0 +1,41 @@
+﻿// <copyright file="ServerTracingInterceptorOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Contrib.Instrumentation.GrpcCore
+{
+    using OpenTelemetry.Context.Propagation;
+
+    /// <summary>
+    /// Options for the ServerTracingInterceptor.
+    /// </summary>
+    public class ServerTracingInterceptorOptions
+    {
+        /// <summary>
+        /// The default propagator.
+        /// </summary>
+        private static readonly TextMapPropagator DefaultPropagator = new TraceContextPropagator();

```

Comment:
All other instrumentations just take the default propagator. Is there a reason why we need a different default here?

Reply:
No, I'll switch the default to Propagators.DefaultTextMapPropagator